### DISCUSSION TITLE
esp8684

### DISCUSSION THREAD

**Discussion by maulionc** [ExpressLRS/ExpressLRS#3052] [General]
Hello 
I'm looking to start ELRS advanture in my side for micro rc
is it possible to use esp8684 because of small package qfn 4*4mm
https://www.espressif.com/sites/default/files/documentation/esp8684_datasheet_en.pdf

Charles

**Comment by pkendall64** [comment]
No, the 8684 is not supported. The 8685 however is supported. The 8685 is also a 4x4mm package. Make sure to use the 8685H4.

**Reply by maulionc** [reply]
hello 
Thanks for you reply.
can you explain how to setup it, where i can choose this chip ?.

**Reply by pkendall64** [reply]
It's an ESP32C3 internally so when building firmware just choose one of the ESP32C3 targets.

**Reply by flashspys** [reply]
@pkendall64 Is there a technical reason why the 8684 cannot be supported even in future, or is it just not done yet?

**Reply by pkendall64** [reply]
8684 is not supported in the Arduino framework yet. Also it's missing the RMT device which we use for DSHOT, RGB LED and potentially other things in future.
